app/src/main/java/com/chess/clock/engine/TimeControlParser.java

```diff
@@ -23,6 +23,7 @@
 public class TimeControlParser {
 
     private static final String TAG = TimeControlParser.class.getName();
+    private static final int DEFAULT_TIME_INDEX = 7;
```

Comment:
This is highly coupled with the list inside `buildDefaultTimeControlsList`.
IMO it would be nice if those were close together so one has easier time remembering to update one when changing another. 
I'm ok with even extract separate file like TimeControlDefaults.java with just the Array and default index.